### PR TITLE
gitignore vim files

### PR DESCRIPTION
**Pull request by archbirdplus**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -12,5 +12,9 @@ gui_layout.zig.zon
 
 test.png
 
+# editor files
+.vimrc
```
Comment (IntegratedQuantum) [inline .gitignore:16]: Shouldn't it ignore all `**/.vimrc`? You had the .vimrc in a subfolder in the other PR.

Comment (archbirdplus) [reply .gitignore:16]: Nope, since the path is not specified it applies to all filenames.

**Comment by archbirdplus** [comment]
I just discovered the global `.gitignore` file, so this PR is unnecessary.

**Comment by IntegratedQuantum** [comment]
What's a global gitignore?

**Comment by archbirdplus** [comment]
There is an option in the `.gitconfig` file to specify some extra rules to apply to every repository, separate from the local `.gitignore`.

**Pull request closed without merging**